### DISCUSSION TITLE
TTS 0.6/0.61 prnunciations is bad

### DISCUSSION THREAD

**Discussion by snoze** [coqui-ai/TTS#1383] [General Q&A]
Just installed new update from 0.5 to 0.61 but now pronunciations is changed a lot. 

For example it can't read 

"We don't post any"
it pronounce it as "We dan't post any" 

Same is true with word "buying"

Everything was fine till 0.5.

Any idea? 

Thank you

**Comment by erogol** [comment]
What is "it"? 

What command did you run?

**Comment by snoze** [comment]
I use same command. Everything was working fine and smooth till version 0.50. After upgrading to new version 0.61 it is giving weird pronunciations for some of the words.

I am using same command

tts --text="We don't post any."

Switching back to version 0.5 is working fine again.  I have no idea.

Thank you very much for your hard work on Coqui-TTS.
